controllers/openshift_controller.go

```diff
@@ -839,6 +839,21 @@ func (r *KataConfigOpenShiftReconciler) processKataConfigDeleteRequest() (ctrl.R
 		}
 	}
 
+	scc := GetScc()
+	if scc == nil {
+		r.Log.Error(err, "error deleting SCC")
```

Comment:
Is it even possible for `GetScc()` to return `nil` ? It seems not and the other user in `createScc()` doesn't check it FWIW.

Reply:
Yes, you are right. I'll change this